sixtyfps_runtime/corelib/graphics/image.rs

```diff
@@ -310,6 +310,35 @@ pub struct LoadImageError(());
 /// let image = Image::from_rgba8(buffer);
 /// ```
 ///
+/// A popular software (CPU) rendering library in Rust is tiny-skia. The following example shows
+/// how to use tiny-skia to render into a [`SharedPixelBuffer`]:
+/// ```no_run
+/// # use sixtyfps_corelib::graphics::{SharedPixelBuffer, Image, Rgba8Pixel};
+/// let mut pixel_buffer = SharedPixelBuffer::<Rgba8Pixel>::new(640, 480);
+///
+/// let mut pixmap = tiny_skia::PixmapMut::from_bytes(
+///     pixel_buffer.make_mut_bytes(), 640, 480
```

Comment:
Maybe pass `pixel_buffer.width()` and .height here to make it copy-paste friendly.

Reply:
Excellent idea, thanks - will do